provisioning/miniprov/hare_mp/main.py

```diff
@@ -963,6 +986,57 @@ def main():
         help='Target directory; defaults to /tmp/hare.',
         action='store')
 
+    sb_sub_parser.add_argument(
+        '--duration',
+        type=str,
+        nargs=1,
+        help='Time bound log collection, like logs for last n days /'
+             ' weeks / months',
+        action='store')
+
+    sb_sub_parser.add_argument(
+        '--size_limit',
+        type=str,
+        nargs=1,
+        help='Limited sized logs',
+        action='store')
+
+    sb_sub_parser.add_argument(
+        '--service',
```

Comment:
Same here should it be --services instead of --service? as per the design document

Reply:
Yes Addressed.